.githooks/pre-commit

```diff
@@ -3,6 +3,21 @@ cd "$(git rev-parse --show-toplevel)"
 ESLINT="client/node_modules/.bin/eslint"
```

Comment:
Good to emulate the check that Kevin does to make sure that one has Pylint installed. In this case I would add a line similar to 
```
PYLINT="path/to/pylint"
```